### PR TITLE
ARM support (ARM32v7, ARM64v8)

### PR DESCRIPTION
**Pull request by rafie**
- Tested on x64, 
- Tested on arm64v8 (64-bit Ubuntu 18),
- Pending testing on RPi3/Raspbian (32-bit).

### CODE REVIEW

**Review by MeirShpilraien — APPROVED** [review]
Please squash before merge :)

**Comment by rafie** [comment]
Sure thing!
Will merge this one after RedisEdge ARM PR goes in.

**Pull request merged**
